homeassistant/components/zwave_js/triggers/event.py

```diff
@@ -251,3 +251,25 @@ def _create_zwave_listeners() -> None:
     _create_zwave_listeners()
 
     return async_remove
+
+
+class EventTrigger(Trigger):
+    """Z-Wave JS event trigger."""
+
+    @classmethod
+    async def async_validate_trigger_config(
+        cls, hass: HomeAssistant, config: ConfigType
+    ) -> ConfigType:
+        """Validate config."""
+        return await async_validate_trigger_config(hass, config)
+
+    @classmethod
+    async def async_attach_trigger(
+        cls,
+        hass: HomeAssistant,
+        config: ConfigType,
+        action: TriggerActionType,
+        trigger_info: TriggerInfo,
+    ) -> CALLBACK_TYPE:
+        """Attach a trigger."""
+        return await async_attach_trigger(hass, config, action, trigger_info)
```

Comment:
The implementation should move to the class, I just did like this for now to keep the diff small.